apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -910,6 +911,9 @@ type perContainerHandler interface {
 	// request. Generally this just does result.Results[i].Error = error
 	// but the Result type is opaque so we can't do it ourselves.
 	SetError(resultIndex int, err *params.Error)
```

Comment:
Since this always seems to be passed a `common.ServerError(err)`, I would change the signature and implementations to receive `errors.Error` and wrap inside the method.

Reply:
agreed